packages/bpk-component-datatable/readme.md

```diff
@@ -0,0 +1,61 @@
+# bpk-component-datatable
+
+> Backpack datatable component (using react-virtualised).
+
+## Installation
+
+```sh
+npm install bpk-component-datatable --save-dev
+```
+
+## Usage
+
+```js
+import React from 'react';
+import BpkDataTable from 'bpk-component-datatable';
+
+const rows = [
+  { name: 'Jose', description: 'Software Engineer' },
+  { name: 'Rolf', description: 'Manager' }
+]
+
+const onRowClick = row => alert(JSON.stringify(row));
+
+export default () => (
+  <BpkDataTable rows={rows} height={200} dir={'rtl'} onRowClick={onRowClick}>
```

Comment:
This requires intervention by the user to handle RTL, most components just do this automatically so this might trip people up